test/functional/mempool_packages.py

```diff
@@ -199,13 +199,17 @@ def run_test(self):
         assert set(mempool1).issubset(set(mempool0))
         for tx in chain[:CUSTOM_ANCESTOR_LIMIT]:
             assert tx in mempool1
-        # TODO: more detailed check of node1's mempool (fees etc.)
-        # check transaction unbroadcast info (should be false if in both mempools)
-        mempool = self.nodes[0].getrawmempool(True)
-        for tx in mempool:
-            assert_equal(mempool[tx]['unbroadcast'], False)
-
-        # TODO: test ancestor size limits
+        # Check more detailed check of node1's mempool
```

Comment:
these comments are not needed(and indented wrong); I think they can just all be removed since the test is self-explanatory

Reply:
Could you please clarify what you meant by "indented wrong"? I believe these lines are intended to follow the same indentation pattern as the rest of the file. If there's a specific indentation style that should be applied here, I'd appreciate your guidance.

Reply:
But I agreed with you. Those lines are self explanatory. I removed the comments.